src/com/complexible/common/csv/CSV2RDF.java

```diff
@@ -121,8 +125,14 @@ public void run() {
 			reader.close();
 			in.close();
 			out.close();
-		}
-		catch (Exception e) {
+		} catch (FileNotFoundException f) {
+			System.out.printf("File not found%n");
+			LOGGER.log(Level.SEVERE, f.getMessage());
+		} catch (IOException io) {
+			LOGGER.log(Level.SEVERE, io.getMessage());
+		} catch (UnsupportedRDFormatException | RDFParseException | RDFHandlerException e) {
+			LOGGER.log(Level.SEVERE, e.getMessage());
+		} catch (Exception e) {
```

Comment:
this is wrong as this would result in fatal errors being handled, program continues to run and prints "Converted %,d rows to %,d triples%n"